test/test_grudge.py

```diff
@@ -991,105 +982,102 @@ def test_bessel(actx_factory):
             b=(1.0,)*dims,
             nelements_per_axis=(8,)*dims)
 
-    discr = DiscretizationCollection(actx, mesh, order=3)
+    dcoll = DiscretizationCollection(actx, mesh, order=3)
+
+    nodes = thaw(actx, op.nodes(dcoll))
+    r = actx.np.sqrt(nodes[0]**2 + nodes[1]**2)
 
-    nodes = sym.nodes(dims)
-    r = sym.cse(sym.sqrt(nodes[0]**2 + nodes[1]**2))
+    # FIXME: Bessel functions need to brought out of the symbolic
+    # layer. Related issue: https://github.com/inducer/grudge/issues/93
+    def bessel_j(actx, n, r):
+        from grudge import sym, bind
+        return bind(dcoll, sym.bessel_j(n, sym.var("r")))(actx, r=r)
```

Comment:
@inducer I've left this test partially converted, but still call out to `sym` to evaluate the Bessel functions. I've linked the related issue #93 here for reference